app/models/ApplicationRoute.scala

```diff
@@ -22,7 +22,7 @@ import scala.language.implicitConversions
 object ApplicationRoute extends Enumeration {
   type ApplicationRoute = Value
 
-  val Faststream, Edip, Sdip = Value
+  val Faststream, Edip, Sdip, SdipFaststream = Value
```

Comment:
I think you might need the ApplicationRoute in admin as well. Otherwise it might break something as I did before.